README.md

```diff
@@ -728,7 +733,7 @@ wiggletools sum test/fixedStep.wig default 10 test/variableStep.wig
 
 * When a set of iterators A1, A2 ... is composed by a n-ary iterator M, M will skip the regions which are skipped by all the input iterators. However, in the presence of more than one input iterators that do not perfectly overlap, there will be regions which are covered by say A1, but not A2. Two behaviours are defined: if M is *strict*, it skips those regions, else it replaces missing values with the corresponding default values. 
 
-By default, n-ary iterators are not strict, but they can be made so with the *strict* keyword after the n-ary function name:
+By default, n-ary iterators are not strict, but they can be made so with the *strict* keyword after the n-ary function name (this is also true of the pearson iterator):
```

Comment:
I think adding a special clause for the pearson (binary|) operator is redundant and break the consistency of the documentation